src/Singular.jl

```diff
@@ -81,7 +81,7 @@ export ZZ, QQ, FiniteField, FunctionField, CoefficientRing, Fp
 const libsingular = Singular_jll.libsingular
 const binSingular = Singular_jll.Singular_path
 
-const libsingula_julia = libsingular_julia_jll.libsingular_julia
+const libsingular_julia = libsingular_julia_jll.libsingular_julia
```

Comment:
`libsingular_julia` is used in
```
src/LibSingular.jl:7:@wrapmodule(libsingular_julia)
```
but this works right now "by accident" because we do `using libsingular_julia_jll` a few lines above, which also defines `libsingular_julia`.

The main impact of this patch then is to make `Singular.libsingular_julia` const, which could have a performance impact, I guess.